pac4j-kerberos/src/main/java/org/pac4j/kerberos/credentials/authenticator/KerberosTicketValidation.java

```diff
@@ -0,0 +1,50 @@
+package org.pac4j.kerberos.credentials.authenticator;
+
+import java.util.HashSet;
+
+import javax.security.auth.Subject;
+import javax.security.auth.kerberos.KerberosPrincipal;
+
+import org.ietf.jgss.GSSContext;

```

Comment:
Where does this class come from? I don't see the dependency in the `pom.xml` file.